apps/backend/config/app_config.ts

```diff
@@ -51,9 +51,54 @@ export default class AppConfig {
   }
 
   getSSLConfig() {
+    let sslKey, sslCert, sslCA;
+
+    if (typeof this.get('DATABASE_SSL_KEY') === 'string') {
+      if (this.get('DATABASE_SSL_KEY')?.indexOf('-BEGIN') !== -1) {
+        sslKey = this.get('DATABASE_SSL_KEY')
+      } else {
+        // Verify file exists
+        if (fs.statSync(this.get('DATABASE_SSL_KEY')!).isFile()) {
```

Comment:
code scanner thing probably throwing a fit over the exclamation mark.  think we can get typescript to accept it if we swap to something like the following:

```suggestion
        if (this.get('DATABASE_SSL_KEY') && fs.statSync(this.get('DATABASE_SSL_KEY')).isFile()) {
```

this change will need to be repeated a few times.

i think it'll also help with the fs.readFileSync call on (current) line 62 since typescript will know that that call will work